tests/acceptance/features/bootstrap/GraphContext.php

```diff
@@ -217,22 +217,39 @@ public function adminHasDeletedGroupUsingTheGraphApi(string $group): void {
 	 * sends a request to delete a user using the Graph API
 	 *
 	 * @param string $user username is used as the id
+	 * @param string|null $userAsAdmin
 	 *
 	 * @return void
 	 * @throws GuzzleException
 	 */
-	public function adminDeletesUserUsingTheGraphApi(string $user): void {
+	public function adminDeletesUserUsingTheGraphApi(string $user, string $userAsAdmin = null): void {
+		$credentials = $this->getAdminOrUserCredentials($userAsAdmin);
+
 		$this->featureContext->setResponse(
 			GraphHelper::deleteUser(
 				$this->featureContext->getBaseUrl(),
 				$this->featureContext->getStepLineRef(),
-				$this->featureContext->getAdminUsername(),
-				$this->featureContext->getAdminPassword(),
+				$credentials["username"],
+				$credentials["password"],
 				$user
 			)
 		);
 	}
 
+	/**
+	 * @When /^the user "([^"]*)" deletes a user "([^"]*)" using the Graph API$/
+	 *
+	 * @param string $userAsAdmin
```

Comment:
this may not be admin user always. please use sth like $actor

Reply:
Renamed the variable name `userAsAdmin` to `byUser`, to make consistent with
```
/**
	 * @When user :byUser tries to add user :user to group :group using the Graph API
	 *
	 * @param string $byUser
	 * @param string $user
	 * @param string $group
	 *
	 * @return void
	 */
	 ```